sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java

```diff
@@ -348,22 +347,17 @@ static RetryJobIdResult getRetryJobId(
     UNKNOWN,
   }
 
-  @VisibleForTesting
-  static TableReferenceProto.TableReference toTableRefProto(TableReference ref) {
-    TableReferenceProto.TableReference.Builder builder =
-        TableReferenceProto.TableReference.newBuilder();
-    if (ref.getProjectId() != null) {
-      builder.setProjectId(ref.getProjectId());
-    }
-    return builder.setDatasetId(ref.getDatasetId()).setTableId(ref.getTableId()).build();
+  static String toProjectResourceName(String projectName) {
+    return "projects/" + projectName;
```

Comment:
Is this format standardized ? If so can we add a reference here ?

Reply:
Done.